go/apps/api/openapi/spec/paths/_internal/vault/decrypt/InternalVaultDecryptRequestBody.yaml

```diff
@@ -0,0 +1,13 @@
+type: object
+required:
+  - keyring
+  - encrypted
+properties:
```

Comment:
Can we just make this an array to support decrypting multiple data pieces? This would help with the listKeys endpoin to just do a single req